docs/reference/tab-widgets/troubleshooting/data/total-shards-per-node.asciidoc

```diff
@@ -0,0 +1,191 @@
+//////////////////////////
+
+[source,console]
+--------------------------------------------------
+PUT my-index-000001
+{
+  "settings": {
+    "index.routing.allocation.total_shards_per_node": "1"
+  }
+}
+
+--------------------------------------------------
+// TESTSETUP
+
+[source,console]
+--------------------------------------------------
+DELETE my-index-000001
+--------------------------------------------------
+// TEARDOWN
+
+//////////////////////////
+
+// tag::cloud[]
+In order to get the shards assigned we'll need to increase the number of shards 
+that can be collocated on a node in order to get all shards assigned. 
+We'll achieve this by inspecting the configuration for the `index.routing.allocation.total_shards_per_node` 
+<<indices-get-settings, index setting>> and increasing the configured value for the
+indices that have shards unassigned.
+
+Note that this requires the current data nodes to have enough disk space to host 
+all the data.
+If the current deployment configuration doesn't have enough capacity to store
+all the data, including the unassigned shards, please contact 
+https://support.elastic.co[Elastic Support] to discuss available options.
```

Comment:
Perhaps "please add nodes or contact support discuss available options"?

Reply:
Same as above. Adding nodes is not available as the cluster is not green.